dashboard/app/views/pd/teachercon1819_registration/partner_submitted.html.haml

```diff
@@ -0,0 +1,10 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+
+%p
+  We’re excited you’re planning to join us this summer! You will receive more information
+  about travel approximately six weeks before TeacherCon. In the meantime, please
+  %u
```

Comment:
This should probably be a strong; the `u` tag is no longer intended to be used just for presentation purposes